wrap/cmake/PybindWrap.cmake.in

```diff
@@ -89,9 +64,9 @@ function(pybind_wrap
   # ~~~
   add_custom_command(OUTPUT ${generated_cpp}
                      DEPENDS ${interface_header}
-                             ${CMAKE_SOURCE_DIR}/wrap/interface_parser.py
-                             ${CMAKE_SOURCE_DIR}/wrap/pybind_wrapper.py
-                             ${CMAKE_SOURCE_DIR}/wrap/template_instantiator.py
+                             @GTWRAP_SOURCE_DIR@/gtwrap/interface_parser.py
+                             @GTWRAP_SOURCE_DIR@/gtwrap/pybind_wrapper.py
```

Comment:
This dynamically generates `PybindWrap.cmake`? Wonder why this is necessary. Can we create this without dynamic codegen?

Reply:
Needed it so that it picks up the correct directory where the `gtwrap` repo is installed in case a project is being wrapped independently.

Reply:
I'll approve but this is kinda scary for newcomers, maybe a bit more documentation on how this process is carried out?

Reply:
No more dynamics codegen! :tada: